[debug] ----------------------------------------------------------------------
[debug] Command:      /usr/local/Cellar/node/7.10.0/bin/node /usr/local/bin/firebase deploy
[debug] CLI Version:  3.9.0
[debug] Platform:     darwin
[debug] Node Version: v7.10.0
[debug] Time:         Mon May 29 2017 18:46:34 GMT-0700 (PDT)
[debug] ----------------------------------------------------------------------
[debug] 
[debug] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token { refresh_token: '1/TZEwC0zETTwfFMp5hryvtRWCeAS9Hyyb8oZNKxgnDLc',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  grant_type: 'refresh_token',
  scope: 'email https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/cloudplatformprojects.readonly https://www.googleapis.com/auth/firebase openid' } 
 Mon May 29 2017 18:46:34 GMT-0700 (PDT)
[debug] <<< HTTP RESPONSE 200
[debug] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/classbook-34b63  
 Mon May 29 2017 18:46:35 GMT-0700 (PDT)
[debug] <<< HTTP RESPONSE 200
[debug] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/classbook-34b63/tokens  
 Mon May 29 2017 18:46:35 GMT-0700 (PDT)
[debug] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'classbook-34b63'...
[info] 
[debug] TypeError: targetNames.join is not a function
    at deploy (/usr/local/lib/node_modules/firebase-tools/lib/deploy/index.js:68:57)
    at Command._action (/usr/local/lib/node_modules/firebase-tools/commands/deploy.js:47:12)
    at /usr/local/lib/node_modules/firebase-tools/lib/command.js:176:29
    at tryCatch (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:539:12)
    at invokeCallback (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:554:13)
    at publish (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:522:7)
    at flush (/usr/local/lib/node_modules/firebase-tools/node_modules/rsvp/dist/rsvp.js:2414:5)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
[error] 
[error] Error: An unexpected error has occurred.
